### ISSUE TITLE
Is it possible to publish the report only with the details of the failed tests?

### ISSUE THREAD

**Issue by jaalvarezvi** [serenity-bdd/serenity-core#2521]

**Comment by wakaleo** [comment]
No, but you can easily filter by failed tests, either from the home page or by typing "failed" in the search filter field.

**Comment by jaalvarezvi** [comment]
Thanks a lot for the quick response. We are facing the following issue. Tests take about 3 min to run in Jenkins, but it takes up to 7 minutes to publish the report. So, I thought that if we publish the report only with the failed tests could reduce this time. Do you maybe have any idea of what can we do? Thank you again.

**Comment by wakaleo** [comment]
7 minutes is WAY too long to generate the reports, something is wrong there. Report generation, even for a very large project, should take under a minute. I would check the jenkins set up to see if there is anything that might be slowing down IO operations.

**Comment by globalworming** [comment]
Report generation should be fast, the report can be quite big though. When you want faster feedback, publish the serenity-summary.html first or send it via email to the persons interested.

**Comment by wakaleo** [comment]
Closed presuming answered.
